tests/utils/normalizeclipboarddata.js

```diff
@@ -17,4 +17,44 @@ describe( 'normalizeClipboardData', () => {
 			normalizeClipboardData( '<span class="Apple-converted-space"> </span>x<span class="Apple-converted-space">\u00a0</span>' )
 		).to.equal( ' x ' );
 	} );
+
+	it( 'should strip all spans with no attributes', () => {
```

Comment:
You're right. The entire #2 is about this. We may lose some data. But, at least for now, it's more important to keep the content clean from non-breaking spaces. Then, we can somehow support copy-pasting them within the editor (they'd need to be marked in a special way when stringified to HTML). And we'll never be able to handle copy-pasting them from outside of the editor because, by converting many normal spaces to non-breaking ones, Chrome loses that data.